assets/sass/modules/content-blocks/_carousel.scss

```diff
@@ -0,0 +1,60 @@
+//--------------------------------------------------------------
+// slide Block
+//--------------------------------------------------------------
+.slide {
+	margin-bottom: 0;
+	overflow-y: hidden;
+	position: relative;
+	z-index: 1;
+
+	// Transparent overlay.
+	&::after {
+		@include position(absolute, 0 0 null null);
+		@include size(100%);
+
+		background-color: rgba($color-black, 0.3);
+		content: '';
+		z-index: 2;
+	} // &:after
+
+	// Content container.
+	&-content {
+		align-items: center;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		margin: 0 auto;
+		max-width: 50%;
+		min-height: rem(500); // height of slide.
```

Comment:
Should we set the `min-height` for the hero and carousel to be the same to give a more consistent presentation?  I know they're being broken out into two separate blocks with this change, but it might be good for them to remain some of that visual consistency since they're made up of the same general elements.

Reply:
Fixed.